contributingGuides/BUGZERO_CHECKLIST.md

```diff
@@ -13,9 +13,8 @@ Source of bug:
   - [ ] 1z. Other:
 
 Where bug was reported:
-  - [ ] 2a. Reported on production
-  - [ ] 2b. Reported on staging (deploy blocker)
-  - [ ] 2c. Reported on both staging and production
+  - [ ] 2a. Reported on production (eg. bug slipped through the normal regression and PR testing process on staging)
+  - [ ] 2b. Reported on staging (eg. found during regression or PR testing)
   - [ ] 2d. Reported on a PR
```

Comment:
```suggestion
  - [ ] 2c. Reported on a PR
```

Reply:
Unless the parser expects this to be d?

Reply:
Heh, this is part of the fun... re-ordering these will mess up all the data parsing that happens on this because all previous options for `2c` were "Reported on both staging and production" and it's really hard to track when this goes live (or I could just go back and update all the parsed data).

I'd like to keep it as 2d if that's OK.